utils/training_utils.py

```diff
@@ -183,7 +183,7 @@ def train(model, num_classes, is_inception, optimizer, scheduler, loss_func, epo
             if is_cyclic_lr:
               scheduler.step()
         
-        correct_preds = torch.where(preds == labels, preds, torch.tensor([-1 for i in preds]).to(device))
+        correct_preds = torch.where(preds == labels, preds, -1)
```

Comment:
No need for the to(device) anymore?

Reply:
Nope, -1 is a scalar so there's no way to put it on the device

Reply:
whoops, i thought that to(device) was on the outside. looks good to me